docs/source/raft_ann_benchmarks.md

```diff
@@ -120,7 +130,7 @@ raft_ivf_pq:
   executable: RAFT_IVF_PQ_ANN_BENCH
   disabled: false
 ```
-`executable` : specifies the binary that will build/search the index. It is assumed to be
+`executable` : specifies the location of the binary that will build/search the index. It is assumed to be
```

Comment:
It's not the location, it's just the name of the binary/executable that's specified here. By default it's only search for in `${CONDA_PREFIX}/bin/ann` or `cpp/build`